bitnami/common/README.md

```diff
@@ -40,38 +40,49 @@ Bitnami charts can be used with [Kubeapps](https://kubeapps.com/) for deployment
+| Helper identifier                           | Description                                                | Expected Input                                                                                                                                                 |
+|---------------------------------------------|------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `common.capabilities.deployment.apiVersion` | Return the appropriate apiVersion for deployment.          | `.` Chart context                                                                                                                                              |
+
+**Warnings**
+| Helper identifier                           | Description                                                | Expected Input                                                                                                                                                 |
+|---------------------------------------------|------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `common.warnings.rollingTag`                | Warning about using rolling tag.                           | `ImageRoot` see [ImageRoot](#imageroot) for the structure.                                                                                                     |
+
+**Secrets**
+| Helper identifier                           | Description                                                | Expected Input                                                                                                                                                 |
+|---------------------------------------------|------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `common.secrets.name`                       | Generate the name of the secret.                           | `dict "existingSecret" .Values.path.to.the.existingSecret "defaultNameSuffix" "mySuffix" "context" $` see [ExistingSecret](#existingsecret) for the structure. |
+| `common.secrets.key`                        | Generate secret key.                                       | `dict "existingSecret" .Values.path.to.the.existingSecret "key" "keyName"` see [ExistingSecret](#existingsecret) for the structure.                            |
```

Comment:
These ones are hard to understand, because they expect too many arguments in the dictionary. I think we should add an example in the `#existingsecret` section.

Reply:
Done!